### PR TITLE
Feature/async icon fingerprinting

### PR DESCRIPTION
**Pull request by srowhani**
**This project uses [semver](http://semver.org), please check the scope of this pr:**

- [ ] #none# - documentation fixes and/or test additions
- [ ] #patch# - backwards-compatible bug fix
- [x] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
- Resolve icon assets async

### CODE REVIEW

File: addon/components/frost-icon.js [hunk 1]
```diff
@@ -50,12 +49,22 @@ export default Component.extend({
...
 
   // == Lifecycle Hooks =======================================================
+  init () {
+    this._super(...arguments)
+    this.get('iconAssets')
+      .register(this)
+      .then(href => {
+        if (!this.isDestroyed && !this.isDestroying) {
+          const icon = this.get('icon')
+          this.set('href', `${href}#${icon}`)
```
Comment (sglanzer-fire) [inline addon/components/frost-icon.js]: As discussed - scheduleOnce('sync'

File: package.json [hunk 2]
```diff
@@ -95,8 +95,9 @@
     "ember-cli-sass": "^5.6.0",
     "ember-cli-string-utils": "^1.0.0",
     "ember-cli-valid-component-name": "^1.0.0",
+    "ember-fetch": "^1.4.0",
```
Comment (sglanzer-fire) [inline package.json:101]: Any pro/con of this versus `ember-network`?

**Comment by sglanzer-fire** [comment]
Approved - really nice improvement, well done!

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/429/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

File: package.json (hunk 2 above)
Comment (srowhani) [reply package.json:101]: It looks like active development has moved to `ember-fetch`

https://github.com/tomdale/ember-network/issues/17

Comment (sglanzer-fire) [reply package.json:101]: Great, thanks

**Pull request merged**